datahub/v1alpha2/datahub.proto

```diff
@@ -218,21 +218,29 @@ message CreateSimulatedSchedulingScoresRequest {
  *
  */
 message UpdatePodsRequest {
-    message PodInfo {
-        bool is_predicted = 1;
+    message UpdatedPod {
+        // namespace and name in containersai.datahub.resource.metadata.v1alpha2.NamespacedName is required
```

Comment:
namespace and name in containersai.datahub.resource.metadata.v1alpha2.NamespacedName are required